packages/std/src/results/submessages.rs

```diff
@@ -196,13 +201,20 @@ impl From<SubMsgResult> for Result<SubMsgResponse, String> {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct SubMsgResponse {
     pub events: Vec<Event>,
+    #[deprecated = "deprecated in the cosmos-sdk in favor of msg_responses"]
     pub data: Option<Binary>,
```

Comment:
Let's remove that in a separate PR instead of deprecating it

Reply:
Oh, but wait a second. If we claum CosmWasm 2.0 contracts run in CosmWasm 1.x chains, we'd remove the only option to get this data.

Reply:
Hmm, true. I remember now, that's the reason why I deprecated it instead of removing.
If we could convert from `data` to `msg_responses` during deserialization, then we could get rid of the field. But I guess that's not possible, since we miss the type url here.

Reply:
I don't want to get too much into the business of trying to interprete and normalize the data we get from the SDK which already changed multiple times over the years.

I think deprecating is good. Let's try to use a more helpful message, like this:


"Deprecated in the Cosmos SDK in favor of msg_responses. If your chain is running on CosmWasm 2.0 or higher, msg_responses will be filled. For older versions, the data field is still needed since msg_responses is empty in those cases."